lab/src/main/ii.go

```diff
@@ -1,22 +1,54 @@
 package main
 
-import "os"
-import "fmt"
-import "mapreduce"
+import (
+	"fmt"
+	"os"
+	"mapreduce"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+)
 
 // The mapping function is called once for each piece of the input.
 // In this framework, the key is the name of the file that is being processed,
 // and the value is the file's contents. The return value should be a slice of
 // key/value pairs, each represented by a mapreduce.KeyValue.
 func mapF(document string, value string) (res []mapreduce.KeyValue) {
 	// Your code here (Part V).
+	rege, _ := regexp.Compile("([a-z]|[A-Z])+")
```

Comment:
[nit] We can use a built-in regex class instead to be more explicit:
```
[[:alpha:]]    alphabetic (== [A-Za-z])
```

Explore built-in regex classes [here](https://golang.org/pkg/regexp/syntax/).